spec/ace.md

```diff
@@ -186,18 +186,18 @@ Sub-units (e.g. decimals, "0.3", or milli-units, "300m") are NOT permissible.
 
 **Parameters:**
 
-* **request** milli-cores that are requested
-* **limit** milli-cores that can be consumed before the kernel temporarily throttles the process
+* **request** cores that are requested
+* **limit** cores that can be consumed before the kernel temporarily throttles the process
 
 ```json
 "name": "resource/cpu",
 "value": {
-  "request": "250",
-  "limit": "500"
+  "request": "250m",
+  "limit": "500m"
 }
 ```
 
-**Note**: a milli-core is the milli-seconds/second that the app/pod will be able to run. e.g. 1000 would represent full use of a single CPU core every second.
+**Note**: a core is the seconds/second that the app/pod will be able to run. e.g. 1 (or 1000m) would represent full use of a single CPU core every second.
```

Comment:
you've removed the word "milli", even though it is implied in `1000m`. I'm wondering whether we should include it somewhere. Perhaps like "e.g. 1 (or 1000m for milli-seconds)" or similar.